ext/misc/amatch.c

```diff
@@ -1405,7 +1405,7 @@ static int amatchBestIndex(
     pIdxInfo->orderByConsumed = 1;
   }
   pIdxInfo->estimatedCost = (double)10000;
-   
```

Comment:
@stelkasouridis : can you please revert to the committed version of this code? It looks like you removed a newline, and we don't really want to commit that change.